src/station_cmd.cpp

```diff
@@ -2094,6 +2094,42 @@ static Vehicle *ClearRoadStopStatusEnum(Vehicle *v, void *)
 	return nullptr;
 }
 
+/**
+ * Check if a drive-through road stop tile can be cleared.
+ * Road stops built on town-owned roads check the conditions
+ * that would allow clearing of the original road.
+ * @param tile road stop tile to check
```

Comment:
`@param` and `@return` tags don't have to be full sentences, but capitalization and punctuation are nice to have. 🙂 

```suggestion
 * @param tile The road stop tile to check.
```